### ISSUE TITLE
Event RSVP's 'people attending' numbers

### ISSUE THREAD

**Issue by harrylouieknight** [opencollective/opencollective#5338]
Just running a few events for a collective I host and noticed something with the tickets/RSVP.

If one person books two tickets it will still only say 'one person going' instead of two. I wondered if it would make sense to display a number based on how many tickets are sold instead of how many individuals have purchased tickets (whether that's just one or five)?

**Comment by SudharakaP** [comment]
This is also requested in the following Fresh-desk ticket; https://opencollective.freshdesk.com/a/tickets/125729

**Comment by SudharakaP** [comment]
For this I suggest the following changes. 

1) Show the correct total count depending on the number of tickets each person booked. 
2) Show how many tickets each registered person booked underneath their name (see below screenshot).

![image](https://user-images.githubusercontent.com/12435965/193359011-04583525-f729-4b58-b08f-42b4c00a5de2.png)

**Comment by Betree** [comment]
> Show the correct total count depending on the number of tickets each person booked.

I agree that this `𝑥 person going` thing should be the count of tickets sold. Or said with SQL:

```sql
SELECT SUM(quantity) FROM "Orders" WHERE ...
```

> 2. Show how many tickets each registered person booked underneath their name.
That seems like a good proposal to make the frontend coherent.
